src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -19,6 +19,7 @@ export enum FeedbackType {
 }
 
 export const STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER = 'SCAFeedbackIdentifier:';
+export const MANUAL_ASSESSMENT_IDENTIFIER = 'MAFeedbackIdentifier:';
```

Comment:
do we really need this? We have FeedbackType.MANUAL for this

Reply:
I removed it :)